client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import EditorRevisionsListHeader from './header';
+import EditorRevisionsListItem from './item';
+import QueryPostRevisions from 'components/data/query-post-revisions';
+import { getPostRevision, getPostRevisions } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getEditorPostId } from 'state/ui/editor/selectors';
+import { isWithinBreakpoint } from 'lib/viewport';
+
+class EditorRevisionsList extends PureComponent {
+	loadRevision = () => {
+		this.props.loadRevision( this.props.selectedRevision );
+	}
+
+	trySelectingRevision() {
+		if (
+			this.props.selectedRevisionId === null &&
+			this.props.revisions.length > 0 &&
+			isWithinBreakpoint( '>660px' )
+		) {
+			this.props.selectRevision( this.props.revisions[ 0 ].id );
```

Comment:
Very subjective, but when I applied this suggestion I didn't really like it. :) Maybe because the usage of `first` is so local, or because of the `const` that gets introduced, requiring a `!!` cast, instead of just relying on `length`.